environment.yml

```diff
@@ -36,6 +36,7 @@ dependencies:
  - sysroot_linux-64
  - xarray
  # For packaging and testing
+ - pybind11>=2.5.0
```

Comment:
needed for `python -m setup.py --version` to work. Could go in the "for running" section, which really should be "for developing" I suppose (or those broken out).